### PR TITLE
chart with one point crashes (#1452)

### PR DESCRIPTION
**Pull request by mnavasiolava**
The problem was in plotGridlines function - data for gridlines (scope.rpipeGridlines) had NaN values.
This happened because plot had only one point and model range had equal xl/xr values. As the result we've got division by zero during calculating element.x percent value in PlotPoint.prototype.applyAxis.
To fix it I suggest increasing the model range if all elements have the same x or y axis coordinates.

### CODE REVIEW

File: core/src/main/web/outputdisplay/bko-plot/plotitems/std/plotpoint.js [hunk 1]
```diff
@@ -79,6 +79,14 @@
         range.yl = Math.min(range.yl, ele.y);
         range.yr = Math.max(range.yr, ele.y);
       }
+      if(range.xl == range.xr){
```
Comment (samccone) [inline core/src/main/web/outputdisplay/bko-plot/plotitems/std/plotpoint.js]: we should use strict equality here

File: core/src/main/web/outputdisplay/bko-plot/plotitems/std/plotpoint.js [hunk 2]
```diff
@@ -79,6 +79,14 @@
...
+        range.xl = range.xl - 0.1;
+        range.xr = range.xr + 0.1;
+      }
+      if(range.yl == range.yr){
```
Comment (samccone) [inline core/src/main/web/outputdisplay/bko-plot/plotitems/std/plotpoint.js]: and here

**Comment by scottdraves** [comment]
:+1:

**Comment by ildipo** [comment]
everyone seems happy so I merge it :-)

**Pull request merged**